.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,10 @@
 <!--
-⚠️ If you do not respect this template, your pull request will be closed.
-⚠️ Your pull request title should be short detailed and understandable for all.
-⚠️ Also, please add a release note file using reno if the change needs to be
-  documented in the release notes.
-⚠️ If your pull request fixes an open issue, please link to the issue.
+⚠️  If you do not respect this template, your pull request will be closed.
+⚠️  Your pull request title should be short detailed and understandable for all.
+⚠️  Also, please add a release note file using reno if the change needs to be documented in the release notes.
+⚠️  If your pull request fixes an open issue, please link to the issue. Use "Fixes #XXXX" if this PR *fully* closes the issue XXXX.  
```

Comment:
There seems to be unrelated changes in these lines?  Mostly I think it's just spacing, but the "if your pull request fixes an open issue" one adds an extra sentence.

(fwiw, I think this stuff here is pretty inconsequential - I'm not certain anybody really reads it. The `CONTRIBUTING.md` is more important, because we can link it to people.)